tests/zenko_tests/create_buckets.py

```diff
@@ -68,13 +69,15 @@ def wait_for_pods(delete, timeout):
                             _log.info('%s is in %s, restarting' % (container.name, container.state.waiting.reason))
                             v1.delete_namespaced_pod(i.metadata.name, K8S_NAMESPACE, body=delete_opt, grace_period_seconds=0)
                         passed = False
-        if not passed:
-            time.sleep(1)
-        else:
-            break
+                        passed_count = 0
+        if passed:
+            if passed_count >= 10:
+                break
+            passed_count = passed_count + 1
+        time.sleep(1)
 
     if not passed:
-      _log.info('Containers have not become ready and %d has elasped' % timeout)
+      _log.info('Containers have not become ready and %d has elasped' % timeout - time.time())
```

Comment:
minor: could add unit: `%d seconds have elapsed`

Reply:
I decided to omit that change completely as I was just using that change for debugging but thank you!